CircuitPython_Pyloton/code.py

```diff
@@ -0,0 +1,59 @@
+from time import time
+import adafruit_ble
+import board
+import pyloton
+
+# PyLint can't find BLERadio for some reason so special case it here.
+ble = adafruit_ble.BLERadio()    # pylint: disable=no-member
+
+display = board.DISPLAY
+
+
+heart = True
+speed = True
+cad = True
+ams = True
+debug = False
```

Comment:
I would make these all caps because they are "constants" used to switch things on and off.